lib/jobs/dell-racadm-tool-job.js

```diff
@@ -4,9 +4,9 @@
 
 var di = require('di');
 
-module.exports = racadmSetBIOSJobFactory;
-di.annotate(racadmSetBIOSJobFactory, new di.Provide('Job.DellRacadm.SetBIOS'));
-di.annotate(racadmSetBIOSJobFactory, new di.Inject(
+module.exports = RacadmToolJobFactory;
```

Comment:
Change RacadmToolJobFactory to racadmToolJobFactory, factory function name should start with lower case.